### DISCUSSION TITLE
go-sdk 1.11.8 introspect type error

### DISCUSSION THREAD

**Discussion by sixleaveakkm** [ory/hydra#3111] [Q&A] [answered]
Thanks to the fix in 1.11.8 of this,
sdk: Correct polymorph type for consent session (#3074) (646459a), closes #3058
the consent session is able to parse `ext` field correctly. 

But it seems a same problem exists in admin introspect,
```go
type OAuth2TokenIntrospection struct {
	...
	// Extra is arbitrary data set by the session.
	Ext map[string]map[string]interface{} `json:"ext,omitempty"`
        ...
}
```
which cause an error saying that it cannot unmarshal string to map[string]interface{} if I set the ext field.

Hope you could consider fix it soon

**Comment by vinckr** [comment, marked as answer]
Hello @sixleaveakkm 
so sorry for the late answer!
Would you be willing to [open a bug report](https://github.com/ory/sdk/issues/new?assignees=&labels=bug&template=BUG-REPORT.yml) in the SDK repo 🙏, that would help us get a fix organized, thanks!
